ui/apps/platform/src/Containers/Vulnerabilities/PlatformCves/PlatformCve/PlatformCvePage.tsx

```diff
@@ -76,8 +82,10 @@ function PlatformCvePage() {
         perPage,
         sortOption,
     });
-    const metadataRequest = usePlatformCveMetadata({ cve: cveId, query, page, perPage });
```

Comment:
I noticed `page` and `perPage` aren't being used. Is that correct?

Reply:
Yes good catch, there have been a couple of cases of this recently due to some refactoring and this one slipped through. I'll push this update before merging.

Reply:
Oh oops I misunderstood, it looks like `page` and `perPage` are already being removed as part of this PR, is that what you meant?

These aren't needed anymore since the `usePlatformCveMetadata` hook no longer requests pagination data as part of the GraphQL query.